src/components/search/utils.js

```diff
@@ -1,3 +1,60 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+import { getSearchFacetFilters } from '@edx/frontend-enterprise-catalog-search';
+import { features } from '../../config';
+
 export function isShortCourse(course) {
   return course.course_length === 'short';
 }
+
+const messages = defineMessages({
+  programsTitle: {
+    id: 'search.facetFilters.programs.title',
+    defaultMessage: 'Program',
+    description: 'Title for the programs facet filter',
+  },
+  programsTypeaheadPlaceholder: {
+    id: 'search.facetFilters.programs.typeahead.placeholder',
+    defaultMessage: 'Find a program...',
+    description: 'Placeholder for the programs typeahead input',
+  },
+  programsTypeaheadAriaLabel: {
+    id: 'search.facetFilters.programs.typeahead.aria.label',
+    defaultMessage: 'Type to find a program',
+    description: 'Aria label for the programs typeahead input',
+  },
+});
+
+export function getSearchFaceFilters(intl) {
```

Comment:
were you mean `getSearchFaceFilters` or `getSearchFacetFilters` ?

Reply:
Done